### PR TITLE
Enable blog post section

### LINKED ISSUE #1696: Enable blog post section

### PR DESCRIPTION
**Pull request by Fashander**
# Description

Closes #1696.

## Readiness checklist

* [ ] I added unit tests for new functionality or bug fixes.
* [ ] I added integration tests for new functionality or bug fixes.
* [ ] I added compatibility tests for new functionality or bug fixes.
* [ ] I made spot refactorings.
* [ ] I updated user documentation.
* [ ] I ran `task all`, and it passed.
* [ ] I added/updated comments for both exported and unexported top-level declarations (functions, types, etc).
* [ ] I checked comments rendering with `task godocs`.
* [x] I ensured that the title is good enough for the changelog.
* [x] (for maintainers only) I set Reviewers ([`@FerretDB/core`](https://github.com/orgs/FerretDB/teams/core)), Assignee, Labels, Project and project's Sprint fields.
* [x] I marked all done items in this checklist.

### CODE REVIEW

**Comment by noisersup** [comment]
Is it ready for review?

**Comment by Fashander** [comment]
@noisersup Not yet

File: website/docusaurus.config.js [hunk 1]
```diff
@@ -66,7 +71,11 @@ const config = {
             position: 'left',
             label: 'Documentation',
           },
-          // {to: '/blog', label: 'Blog', position: 'left'},
+          // {
```
Comment (Fashander) [inline website/docusaurus.config.js:74]: @AlekSi  Here, I need to remove the comment around the blog but can't do that without getting errors. We need the default content plugin [here](https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-content-blog) to use the blog.

Comment (AlekSi) [reply website/docusaurus.config.js:74]: Please send PR to https://github.com/FerretDB/deps to add that dependency

Comment (AlekSi) [reply website/docusaurus.config.js:74]: More specifically, that file should be updated: https://github.com/FerretDB/deps/blob/main/docusaurus-docs/package.json

Comment (Fashander) [reply website/docusaurus.config.js:74]: Looking through again, I think this dependency already exists, it comes with this one `@docusaurus/preset-classic`. At the moment, I think there's no way to route to the path: `website/blog`.  Is it possible the problem is from there? 

It worked when I added this `- ./website/blog:/workdir/docusaurus-docs/blog:ro` to the Docker compose file [here](https://github.com/FerretDB/FerretDB/blob/main/docker-compose.yml?plain=1#L129-L135).

Comment (AlekSi) [reply website/docusaurus.config.js:74]: Docker Compose configuration change looks correct, please add it to this PR

Comment (Fashander) [reply website/docusaurus.config.js]: PTAL

File: website/blog/2022-12-28-database-alternatives-mongodb-2023.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+authors: [alex]
+image: https://i.imgur.com/ejygHXM.png
+tags: [mongodb, document database]
+---
+
+The top 5 MongoDB-compatible alternatives to MongoDB include: FerretDB, DocumentDB, CosmosDB, GaussDB(for Mongo), and ToroDB.
+
+<!--truncate-->
+
+![5 alternatives to mongoDB](https://i.imgur.com/ejygHXM.png)
```
Comment (AlekSi) [inline website/blog/2022-12-28-database-alternatives-mongodb-2023.md:13]: We should store images in our repo using Git LFS

**Review by AlekSi — COMMENTED** [review]
I think it is better to separate PR that enables blog from PR that adds a new blog post, as they should be reviewed differently

**Review by rumyantseva — APPROVED** [review]
LGTM overall (`git lfs` comment makes sense to me too, yes).

Looking forward to seeing our new blog platform! 🎉

**Review by AlekSi — COMMENTED** [review]
CI fails

**Review by AlekSi — APPROVED** [review]

**Pull request merged**